src/services/functionAppService.ts

```diff
@@ -138,16 +137,16 @@ export class FunctionAppService extends BaseService {
     return await this.get();
   }
 
-  private async uploadZippedArfifactToFunctionApp(functionApp) {
+  public async uploadZippedArfifactToFunctionApp(functionApp: Site, functionZipFile: string) {
```

Comment:
why switching to `public`? Needed?

Reply:
It's used inside the rollback service now. to re-deploy function zip to the function app